app/ai/voice/agents/breeze_buddy/managers/pre_checks.py

```
@@ -0,0 +1,429 @@
+                            return _apply_default(
+                                pre_check,
+                                f"API returned status {status_code}",
+                            )
+                        last_error = f"API returned status {status_code}"
+
+            except aiohttp.ClientError as e:
+                last_error = f"HTTP client error: {e}"
+                logger.warning(
+                    f"Pre-check '{name}': {last_error} (attempt {attempt}/{http_config.max_retries})"
+                )
+            except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)